packages/teleport/src/Desktops/Desktops.tsx

```diff
@@ -53,7 +61,7 @@ export function Desktops(props: State) {
         </Box>
       )}
       {attempt.status === 'failed' && <Danger>{attempt.statusText}</Danger>}
-      {attempt.status === 'success' && (
+      {hasDesktops && (
```

Comment:
should desktop also have a `view documentation` button similar to our kube view? (since they both have no "add" directions yet)

Reply:
Sure, added